sunpy/util/util.py

```diff
@@ -179,12 +186,13 @@ def replacement_filename(path):
                 return newpath
 
 
-#==============================================================================
+# =============================================================================
 # expand list from :http://stackoverflow.com/a/2185971/2486799
-#==============================================================================
+# =============================================================================
 def expand_list(input):
```

Comment:
Does this function work on, or need to work on, arbitrarily nested lists?  Googling around, it seems there are many possible implementations of this functionality.  Also, where is it used in sunpy or in this PR?  I couldn't find it used anywhere.